pmd-java/pom.xml

```diff
@@ -183,6 +183,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This dependency needs to be added to each module (only done yet for pmd-apex and pmd-java). Otherwise IntelliJ won't find the tests.

When building with maven, this is not needed, since the surefire-plugin has these dependency explicit added (junit-jupiter-platform), but eventually, we maybe can remove this extra config for surefire.